cmd/containerd-shim-runhcs-v1/main.go

```diff
@@ -44,17 +43,6 @@ var (
 	idFlag string
 )
 
-func stack() []byte {
-	buf := make([]byte, 1024)
-	for {
-		n := runtime.Stack(buf, true) // true means all goroutines
-		if n < len(buf) {
-			return buf[:n]
-		}
-		buf = make([]byte, 2*len(buf))
-	}
-}
-
```

Comment:
Instead of getting rid of this why don't we just actually use it in `DiagStacks`? Just need to up the buffer to 4k

Reply:
We already do something similar for `DiagStacks` I don't think this is necessary.

Reply:
That's what I'm saying, why don't we just call this and get rid of the stack logic that's inline in that function.

Reply:
I'm gonna say that's out of scope here :) If we want to improve the `DiagStacks` code, let's do it outside of this PR.